phow-comps.md

```diff
@@ -0,0 +1,39 @@
+
+Prepare the foam board: Cut the foam board into two pieces that are roughly the same size as the wings of your paper airplane.
+
+Attach the servo motor: Use the lightweight wire to attach the servo motor to one of the foam board pieces. Make sure the servo is centered on the foam board and that the wire is securely attached.
+
+Install the radio transmitter and receiver: Follow the instructions provided with your radio-controlled aircraft kit to install the transmitter and receiver onto the foam board.
+
+Attach the battery pack: Use the wire to connect the battery pack to the transmitter and receiver. Make sure the battery pack is securely attached to the foam board.
+
+Attach the foam board to the paper airplane: Use the lightweight wire to attach the foam board pieces to the wings of your paper airplane. Make sure the foam board is evenly balanced on the wings and that the wire is securely attached.
+
+Test the RC paper airplane: Follow the instructions provided with your radio-controlled aircraft kit to test the RC paper airplane. Make any necessary adjustments to the servo motor or radio transmitter and receiver to ensure the paper airplane is responding correctly.
+
+## Im going to be buying 4 of the kits which i can assemble myself, since theres a high chance for things to break with this project. I will also be buying templates to make 3d printed airplanes to which i can attach the kit and make them fly.
+
+## Budget including shipping and taxes(Aud)
+
+| Product         | Supplier/Link                         | Cost(aud)   |
+| --------------- | ------------------------------------- | ------ |
+|Power Up 4.0 |hhttps://www.amazon.com.au/POWERUP-4-0-Next-Generation-Smartphone-Stabilizer/dp/B08JLZVB3Z/ref=sr_1_2?crid=28KSRGXSIJ574&keywords=powerup&qid=1671603116&s=amazon-devices&sprefix=power%2Camazon-devices%2C481&sr=1-2| $76|
+|Power Up 4.0 |hhttps://www.amazon.com.au/POWERUP-4-0-Next-Generation-Smartphone-Stabilizer/dp/B08JLZVB3Z/ref=sr_1_2?crid=28KSRGXSIJ574&keywords=powerup&qid=1671603116&s=amazon-devices&sprefix=power%2Camazon-devices%2C481&sr=1-2| $76|
+|Power Up 4.0 |hhttps://www.amazon.com.au/POWERUP-4-0-Next-Generation-Smartphone-Stabilizer/dp/B08JLZVB3Z/ref=sr_1_2?crid=28KSRGXSIJ574&keywords=powerup&qid=1671603116&s=amazon-devices&sprefix=power%2Camazon-devices%2C481&sr=1-2| $76|
+|Power Up 4.0 |hhttps://www.amazon.com.au/POWERUP-4-0-Next-Generation-Smartphone-Stabilizer/dp/B08JLZVB3Z/ref=sr_1_2?crid=28KSRGXSIJ574&keywords=powerup&qid=1671603116&s=amazon-devices&sprefix=power%2Camazon-devices%2C481&sr=1-2| $76|
+|Power Up templates| https://www.amazon.com.au/s?k=powerup&ref=nb_sb_noss#:~:text=PowerUp%20Paper%20Airplane%20Templates%20Models%20%2D%2012%20Paper%20Airplane%20Templates%20for%20The%202%2C0%2C%203.0%2C%20%26%204.0.%20Includes%20Color%20Templates%20%26%20Display%20Stand%20Models.%20for%20Hobbyists%20and%20Tinkerers | $27 |
```

Comment:
hiii is this the correct link? thank you sm!

https://www.amazon.com.au/s?k=powerup&ref=nb_sb_noss#:~:text=PowerUp%20Paper%20Airplane%20Templates%20Models%20%2D%2012%20Paper%20Airplane%20Templates%20for%20The%202%2C0%2C%203.0%2C%20%26%204.0.%20Includes%20Color%20Templates%20%26%20Display%20Stand%20Models.%20for%20Hobbyists%20and%20Tinkerers